rispy/writer.py

```diff
@@ -101,13 +182,14 @@ def dump(references: List[Dict], file: TextIO, mapping: Optional[Dict] = None):
     Args:
         references (List[Dict]): List of references.
         file (TextIO): File handle to store ris formatted data.
-        mapping (Dict, optional): Custom RIS tags mapping.
+        implementation (RisImplementation): RIS implementation; base by
+                                            default.
     """
-    text = dumps(references, mapping)
+    text = dumps(references, implementation=implementation)
```

Comment:
`**kw` arguments should also be passed.

Reply:
Apparently, this is not covered by a test.

Reply:
nice catch, I'll add a test case as well

Reply:
Great